pom.xml

```diff
@@ -53,16 +53,16 @@
         <gravitee-cloud-initializer.version>2.1.0</gravitee-cloud-initializer.version>
         <gravitee-spec-gen-api.version>1.1.0</gravitee-spec-gen-api.version>
         <gravitee-common.version>4.6.0</gravitee-common.version>
-        <gravitee-secret-api.version>1.0.0-alpha.3</gravitee-secret-api.version>
+        <gravitee-secret-api.version>1.0.0</gravitee-secret-api.version>
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Config Generic   
**Description:** Generic Config Credential   
**Severity:** HIGH   
**Match:**         <gravitee-secret-api.version>*****</gravitee-secret-api.version>  
[This comment was created by Aqua Pipeline]